src/variety/slither.js

```diff
@@ -160,6 +232,25 @@
 		}
 	},
 
+	"Graphic@swslither": {
+		drawSheepWolf: function() {
+			this.vinc("cell_number_image", "auto");
+			var g = this.context;
+			var clist = this.range.cells;
+			for (var i = 0; i < clist.length; i++) {
+				var cell = clist[i];
+				if (cell.qnum >= 5) {
+					g.vid = "cell_text_" + cell.id;
+					g.vhide();
+					var text = cell.qnum === 5 ? "🐑" : "🐺";
```

Comment:
Generally speaking we avoid using emojis within the puzzle canvas, since they look wildly inconsistent across platforms.

Reply:
At first, I was going to copy the way it was done in the Goat Wolf puzzle but found that so ugly. Emojis seemed like the easiest way to get beautifully rendered scalable images. In this instance, I felt like it was appropriate.

I understand your concern about cross-platform inconsistency, but really that only concerns style. The emoji standard would ensure that it would never look broken or anything.

If it's something you can't look past what would you like me to do? Stay consistent with the Goat Wolf puzzle? Use some other bitmap images? Personally, I would rather embed SVGs for the Apple version of the emojis. I could also submit a PR to change the Goat Wolf.

Reply:
Yes, emojis are blocking.

I'm not a fan of the Goats & Wolves bitmaps either, but our graphics library doesn't currently support embedding SVGs. I'd happily remove those bitmaps if there was a PR adding SVG support (which isn't gonna be easy, since the puzzle canvas can be rendered as either inline SVG or as a `<canvas>` element)

For now, you can decide if you want to change it to bitmaps, or draw the symbols yourself. Having less detail would be fine, e.g. just drawing the letters S and W or drawing a few circles.

Reply:
will add some bitmap images later this evening

Reply:
didn't get around to this the other day, but I've now removed the emoji and added some images.